components/IssueCard.vue

```diff
@@ -0,0 +1,15 @@
+<script lang="ts" setup>
+defineProps<{ id: string; title: string }>()
+</script>
+
+<template>
+  <li
+    class="flex flex-col gap-1 rounded-[4px] bg-rad-background-float px-2 pb-6 pt-2 hover:bg-rad-fill-float-hover"
```

Comment:
`hover:bg-rad-fill-float-hover` is invisible in light mode

https://github.com/cytechmobile/radicle-planning-boards/assets/7542054/adcb329d-8a65-4786-a8ca-ac16827625ee

We need either a different color that we use for both (ideally one that is semantically correct given its title) if there is one or specify a different one for `hover:`. Perhaps with can use a semantically correct color but play around with its transparency?

Reply:
I referenced the [radicle issues page](https://app.radicle.xyz/nodes/seed.radicle.xyz/rad:z4V1sjrXqjvFdnCUbxPFqd5p4DtH5/issues) for this, and it seems it suffers from the same issue.

While I agree with you, I also feel like this is going out of scope of the prototype (function over form).

I will leave it as is for now, but if you feels strongly about it, I'll look into it.